invenio_communities/views/api.py

```diff
@@ -177,9 +177,12 @@ def get(self, community_id):
         community = Community.get(community_id)
         if not community:
             abort(404)
-        return self.make_response(
+        etag = community.version_id
+        self.check_etag(etag)
+        response = self.make_response(
                 community, links_item_factory=default_links_item_factory)
```

Comment:
@JavierDelgadoFernandez I know that you haven't touched this line but it's also wrongly indented. Can you fix it too?